_opcodes/op188-bgee.html

```diff
@@ -0,0 +1,20 @@
+---
+n: 188
+name: "Spell Effect: Aura Cleansing"
+param1: "Irrelevant"
+param2: "Stat Value"
+bg1: 0
+bg2: 0
+bgee: 1
+iwd1: 0
+iwd2: 0
+pst: 0
+---
+Set the 'Stat Value' field to any non-zero value if you want no time needed between spells.
+<br/><br/>The effect modifies <a href="../files/ids/bgee/stats.htm#76">STAT #76</a> to 'param2'.
+
+{% capture note %}
+The 'Stat Value' parameter can take any value. If you set it to 0, you will disable the effect (for its duration).
+{% endcapture %}
+
+{% include note.html %}
```

Comment:
This sounds identical to the old version — no need for a separate file. The note is very generic, since that method applies to most (if not all) numeric effects.

Reply:
> no need for a separate file.

I added a separate file because of the anchor to the relevant STAT in the EE version of `STATS.IDS`...

Reply:
They didn't move the stats, so it's not really important that it's linking to the ee listing instead of the individual ones. They'll all get merged eventually anyway.

Reply:
Got it, will fix...